tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -905,6 +905,23 @@ function foo(
 $example = function () use ($message1,$message2) {
 };',
             ],
+            'test first element in same line, space before comma and inconsistent indent' => [
+                '<?php foo(
+    "aaa
+    bbb",
+    $c,
+    $d,
+    $e,
+    $f
+);
+',
+                '<?php foo("aaa
```

Comment:
can we add similar test case but with different types of comments around?

Reply:
@keradus done